src/Joblify.Core.Data/Models/User.cs

```diff
@@ -14,7 +14,7 @@ public class User
 
         [Required]
         [ForeignKey("ExternalProvider")]
-        public byte ExternalProviderId { get; set; }
+        public int ExternalProviderId { get; set; }
```

Comment:
Why we need "int" instead "byte". We have only some ExternalProviderId in this case "2"

Reply:
Most of the time I use int. Not for performance but for simplicity. In my honest opinion size like this doesnt matter at all. I can change it if rest of the team want it :)

Reply:
Adding more providers like Twitter, github, whatever else, will make byte unusable anyway

Reply:
let's leave int for now, no point in limiting Id range anyway, not in project this size

Reply:
@Bartosz97 your final thoughts on this?

Reply:
also @jacek15157  @Astenna  @JJMPdsk, what do you think?

Reply:
As you like. This isn't an important thing in our project.